railties/test/commands/notes_test.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rails/command"
+require "rails/commands/notes/notes_command"
+
+class Rails::Command::NotesTest < ActiveSupport::TestCase
+  setup :build_app
+  teardown :teardown_app
+
+  test "`rails notes` displays results for default directories and default annotations" do
+    app_file "app/controllers/some_controller.rb", "# OPTIMIZE: note in app directory"
+    app_file "config/initializers/some_initializer.rb", "# TODO: note in config directory"
+    app_file "db/some_seeds.rb", "# FIXME: note in db directory"
+    app_file "lib/some_file.rb", "# TODO: note in lib directory"
+    app_file "test/some_test.rb", 1000.times.map { "" }.join("\n") << "# FIXME: note in test directory"
```

Comment:
Got curious here and experimented in my console. `1000.times.map { "" }.join("\n")` could be cut down to just `"\n" * 1000`.

So this works

```ruby
app_file "test/some_test.rb", "\n" * 1000 << "# FIXME: note in test directory"
```

Reply:
You're right. I just copied some test cases from the `rake notes` tests 😅. To be real I could also cut it down to just a 2 or 3 digits number since it's just for testing the indentation/alignment of the line number. I'll put up a quick PR for it since it's a little overkill right now.